src/core/backend/co_spawn.cpp

```diff
@@ -27,6 +27,8 @@ plugin_manager::plugin_manager()
     console.log_item("status", "done appending init tabs!");
     console.log_item("status", "initializing python...");
 
+    Py_Initialize();

```

Comment:
Py_Initialize() shouldn't be called. it is already called on L:63 with config. This is most likely why you are having to manually specify paths.